src/spacer/spacer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+type Size = "xxs" | "xs" | "s" | "m" | "l" | "xl" | "xxl";
```

Comment:
```suggestion
const sizes = {
	xxs: "h-[5px]",
	xs: "h-[10px]",
	s: "h-[20px]",
	m: "h-[30px]",
	l: "h-[60px]",
	xl: "h-[90px]",
	xxl: "h-[180px]",
} as const;
```
Style nitpick, but I think this is a bit cleaner.